java/spark-2.0.0/utils.scala

```diff
@@ -0,0 +1,19 @@
+package sparklyr
+
+import org.apache.spark.sql._
+
+object Utils200 {
```

Comment:
@javierluraschi the thought here was that there may be more new functions down the road in later versions of Spark, and we might not want to create a new object for each function, hence the versioning. However, it seems like the collection of methods have been stable enough, so we're probably OK calling this `Repartition`. You're fine with having versioned directory names though?